goth/configuration.py

```diff
@@ -168,6 +171,24 @@ def read_volumes_spec(self) -> Dict[Path, str]:
                 volumes[source] = dest
         return volumes
 
+    def read_environment(self) -> Dict[str, str]:
+        """Read a specification of environment variables from this parser's document.
+
+        The variables must be strings in the form: "{NAME}={VALUE}".
+        """
+        self.ensure_type(list)
+        environment = {}
+        for env_var in self:
+            name_value_pair = env_var.split("=")
+            if len(name_value_pair) != 2:
+                raise ConfigurationParseError(
+                    f"Invalid format of environment variable {env_var}, "
+                    f"expected `VAR_NAME=VAR_VALUE`."
+                )
```

Comment:
That's a bit too restrictive, why not allow entries like: 
```
"MAGIC_KEY=n&gjku89tg#54t=!r4wvrfksg"
```
It's just a matter of replacing `env_var.split("=")` with `env_var.split("=", 1)`.

Reply:
Good one! Addressed it in ec894cf.